test/helpers/constants/images.go

```diff
@@ -16,7 +16,9 @@ package constants
 
 const (
 	// NetperfImage is the Docker image used for performance testing
-	NetperfImage = "docker.io/tgraf/netperf:v1.0"
+	// NB: this image includes netperf and a utility named xping that works
+	// like ping but it also allows to specify the ICMP id.
+	NetperfImage = "quay.io/kkourt/netperf:xping"
```

Comment:
Note for reviewers: we might want to modify the location/name of the image. Suggestions welcome.

Reply:
Yep, we definitely need to store this somewhere else, with sources and Dockerfile.
/cc @aanm

Reply:
We already have this repository https://github.com/cilium/netperf, let's push it there.

Reply:
There are two different things here, the code and the docker image.

For the code, I don't think the netperf repository makes sense, since this is not netperf (it's a modified version of busybox's ping). I think a specific branch on cilium/busybox is probably a better choice.

Is the image location under quay.io/kkourt OK then?

Reply:
After discussing this offline, we came up with https://github.com/cilium/busybox and  quay.io/cilium/net-test:v1.0.0

Reply:
Whoa! You're already planning for minor and patch releases! :smile: